contrib/drivers/filesharedrivers/oceanstor/oceanstor.go

```diff
@@ -0,0 +1,702 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package oceanstor
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/golang/glog"
+	. "github.com/opensds/opensds/contrib/drivers/utils/config"
+	model "github.com/opensds/opensds/pkg/model"
+	pb "github.com/opensds/opensds/pkg/model/proto"
+	"github.com/opensds/opensds/pkg/utils/config"
+	"github.com/satori/go.uuid"
+)
+
+func (d *Driver) Setup() error {
+	if d.Client != nil {
+		return nil
+	}
+
+	var err error
+
+	d.IniConf()
+	cli, err := newRestCommon(d.Config)
+	if err != nil {
+		msg := fmt.Sprintf("get new client failed: %v", err)
+		log.Error(msg)
+		return errors.New(msg)
+	}
+
+	d.Client = cli
+	log.Info("get oceanstor client successfully")
+
+	return nil
+}
+
+func (d *Driver) IniConf() {
```

Comment:
Please rename this method to `InitConf`.

Reply:
Done.